style.css

```diff
@@ -500,18 +440,112 @@
   gap: 20px;
 }
 
-@media screen and (max-width: 700px) {
-  .section1,
-  .work {
-    padding-left: 30px;
-  }
+.contact {
+  background-color: #e5e5e5;
+}
 
-  .about {
-    padding: 4rem 0;
-  }
+.cont {
+  padding: 8rem 2.5rem;
+}
 
-  .about-line {
-    padding: 0 30px;
-    padding-bottom: 10rem;
-  }
+.contact-text {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.2rem;
+  line-height: 3rem;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 5rem;
+}
+
+.input {
+  font-size: 1rem;
+  padding: 0.8rem 1rem;
+  border-radius: 3px;
+  background-color: #e5e5e5;
+  border: none;
+  font-family: inherit;
+  border-bottom: 2px solid #dbd8d7;
+  width: 100%;
+  color: inherit;
+  display: block;
+}
+
+.input:focus {
+  outline: none;
+  box-shadow: 0 1.5rem 2.5rem rgb(0 0 0 / 10%);
+  border-bottom: 4px solid rgb(255 107 0);
+}
+
+.input:focus:invalid {
+  border-bottom: 4px solid #7ed56f;
+}
+
+.label {
+  display: block;
+  font-size: 1rem;
+  font-weight: 700;
+  margin-left: 0.8rem;
+  margin-top: 1rem;
+  transition: all 0.5;
+}
+
+.input:placeholder-shown + .label {
+  opacity: 0;
+  visibility: hidden;
+  transform: translateY(-4px);
+}
+
+.area {
+  margin-top: 15px;
+  width: 100%;
+  border-radius: 3px;
+  background-color: #fbf8f7;
+  border: none;
+  font-family: inherit;
+}
+
+.submit {
+  padding: 15px 12px;
+  background-color: #ff6b00;
+  text-align: center;
+  border: none;
+  margin: 50px 28%;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 22px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  border-radius: 12px;
+  box-shadow: 0 2.5rem 3.5rem rgb(27 27 27 / 10%);
+  color: #fff;
+}
+
+.last {
+  background-color: #e5e5e5;
+  padding: 25px 30%;
+  padding-top: 40px;
+  border-top: 3px solid #dfe1e6;
+}
+
+.forms {
+  padding: 0;
+}
+
```

Comment:
- According to the project requirements, your form section should be displayed using the `flex` property, please try to fix this 🙏 🙏